app/services/users.service.js

```diff
@@ -0,0 +1,57 @@
+import uuid from 'react-uuid';
+import { User } from '../data_access';
+import { Op } from 'sequelize';
+import { MESSAGES } from '../const';
+
+const errorNotFoundMsg = MESSAGES.errors.notFound;
+const getMatchById = async (id) => await User.findByPk(id);
+
+export const getUserList = async (substring = '', limit = 10) => (
+    User.findAll({
+        where: {
+            [Op.and]: [{
+                login: {
+                    [Op.substring]: substring
+                }
+            }, {
+                is_deleted: {
+                    [Op.eq]: false
+                }
+            }]
+        },
+        order: [
+            ['login', 'ASC']
+        ],
+        limit
+    })
+);
+
+export const getUser = async (id) => {
+    const user = await getMatchById(id);
+    if (user && !user.is_deleted) {
```

Comment:
it's better to have sql query with is_deleted: true conditions

Reply:
for example, you can add one more argument to `getMatchByID` with additional conditions

Reply:
Done